Library/Formula/task-spooler.rb

```diff
@@ -5,6 +5,8 @@ class TaskSpooler < Formula
   url 'http://vicerveza.homeunix.net/~viric/soft/ts/ts-0.7.4.tar.gz'
   sha1 '92813a3b0eedfe1d4a177727122e6d08695f6bc8'
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project (if you haven't already) and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!